ant-evm/src/amount.rs

```diff
@@ -67,7 +67,7 @@ impl AttoTokens {
 
     /// Converts the Nanos into bytes
     pub fn to_bytes(&self) -> Vec<u8> {
-        self.0.as_le_bytes().to_vec()
+        self.0.to_be_bytes::<32>().to_vec()
```

Comment:
Really network type endianness (most significant bit first) The 32 states we want exactly 32bytes (256 bits)
Otherwise we risk returning a variable length vec, well we would if using different types at least. So here we force the 32bytes. 

In short we guarantee a 32byte network endian number here really.